monit/jirasync_monit.conf

```diff
@@ -0,0 +1,9 @@
+set daemon 60 #check services ever 60 seconds
+set logfile /var/log/monit.log
+
+check process jirasync with match jirasync every 1 cycles
+    start program = "/usr/bin/systemctl start jirasync.service" with timeout 30 seconds
+    stop program = "/usr/bin/systemctl stop jirasync.service"
+    if 2 restarts within 4 cycle
+        then exec "/bin/bash -c /etc/jirasync/send_email.sh"
+    if 4 restarts within 6 cycles then timeout
```

Comment:
@rochacbruno Just added simple monitoring and alert. we can add more complex alerts if we want soon later.